woocommerce-transbank/libwebpay/HealthCheck.php

```diff
@@ -83,7 +83,7 @@ private function getValidateCertificates() {
 
     // valida version de php
     private function getValidatephp(){
-        if (version_compare(phpversion(), '7.4', '<') and version_compare(phpversion(), '5.5.0', '>=')) {
+        if (version_compare(phpversion(), '7.5', '<') and version_compare(phpversion(), '5.5.0', '>=')) {
```

Comment:
As the plugin works with any version of PHP above PHP 5.5 is better to use this.  
```suggestion
        if (version_compare(phpversion(), '5.5.0', '>=')) {
```